examples/local_transactions/local_transactions.bal

```diff
@@ -0,0 +1,84 @@
+import ballerina/mysql;
+import ballerina/io;
+
+function main(string... args) {
+
+    endpoint mysql:Client testDBEP {
+        host:"localhost",
+        port:3306,
+        name:"testdb",
+        username:"root",
+        password:"root",
+        poolOptions:{maximumPoolSize:5}
+    };
+
+    // Create the tables required for the transaction.
+    var ret = testDBEP->update("CREATE TABLE IF NOT EXISTS CUSTOMER (ID INT, NAME VARCHAR(30))");
+    match ret {
+        int retInt => io:println("CUSTOMER table create status in DB:" + retInt);
+        error err => {
+            handleError("CUSTOMER table Creation failed:", err, testDBEP);
+            return;
+        }
+    }
+    ret = testDBEP->update("CREATE TABLE IF NOT EXISTS SALARY (ID INT, MON_SALARY FLOAT)");
+    match ret {
+        int retInt => io:println("SALARY table create status in DB:" + retInt);
```

Comment:
this should be "DB: " (there should be a space after ":"). Fix in all the places.